tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -232,3 +241,58 @@ def teardown():
+    cmd = (
+        f"oc mirror --config {imageset_config_file} "
+        f"docker://{config.DEPLOYMENT['mirror_registry']} "
+        "--workspace file://oc-mirror-workspace/results-files --v2"
+    )
+
+    try:
+        exec_cmd(cmd, timeout=18000)
+    except CommandFailed as e:
+        # if itms is configured, the oc mirror command might fail (return non 0 rc),
+        # but we want to continue to try to mirror the images manually with applied the itms rules
+        log.warning(f"oc mirror command failed: {e}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)